autoload/neomake/makers/ft/python.vim

```diff
@@ -83,6 +83,11 @@ function! neomake#makers#ft#python#Flake8EntryProcess(entry) abort
     else
         let type = ''
     endif
+    let l:token = matchstr(a:entry.text, "'.*'")
+    if strlen(l:token)
+        let a:entry.length = strlen(l:token) - 2 " subtract the quotes
+    endif
```

Comment:
Length will indeed only be set if text is in quotes (indicating a specific word). It's not full proof but it works quite well for flake8 errors. I've been using this branch for a while now and the only error for which it is wrong is the unused import one.

Length can be set optionally, because not all linters/makers will be able to support it. However, for ones that do it can be really useful. I'm mostly working on python projects with flake8 so I added support for that one, allowing possible additions for other makers to be done later.